DigitalLearningSolutions.Data/Services/TutorialContentService.cs

```diff
@@ -206,7 +209,8 @@ FROM CustomisationTutorials
                          AND Sections.SectionID = @sectionId
                          AND Tutorials.TutorialId = @tutorialId
                          AND Customisations.Active = 1
-                         AND CustomisationTutorials.Status = 1;",
+                         AND CustomisationTutorials.Status = 1
+                         AND Tutorials.ArchivedDate IS NULL;",

```

Comment:
What about if the section is archived?

Reply:
Yep, I hadn't thought of that! Fixed in #154 🙂